src/Main.java

```diff
@@ -1,5 +1,65 @@
 public class Main {
     public static void main(String[] args) {
         System.out.println("Hello world!");
+
+        System.out.println("задача 1");
+
+        for (int i = 1; i <= 10; i++) {
+            System.out.println(i);
+        }
+
+        System.out.println("задача 2");
+
+        for (int e = 10; e >= 1; e--) {
+            System.out.println(e);
+        }
+        System.out.println("задача 3");
+
+        for (int r = 0; r < 17; r = r + 2) {
+            System.out.println(r);
+        }
+        System.out.println("задача 4");
+
+        for (int f = 10; f >= -10; f--) {
+            System.out.println(f);
+        }
+        System.out.println("задача 5");
+
+        for (int d = 1904; d <= 2096; d = d + 4) {
+            System.out.println(d + " год является високосным");
+        }
+        System.out.println("задача 6");
+
+        for (int i = 7; i <= 98; i = i + 7) {
+            System.out.println(i);
+        }
+        System.out.println("задача 7");
+
+        for (int o = 1; o <= 512; o = o * 2) {
+            System.out.println(o);
+        }
+        System.out.println("задача 8");
+
+        int moneyBox = 29000;
+        int total = 0;
+        for (int i = 1; i <= 12; i++) {
+            total = total + moneyBox;
+            System.out.println("Месяц " + i + ", сумма накоплений равна " + total + " рублей");
+        }
+        System.out.println("задача 9");
+
+        int moneyBank = 29000;
+        total = 0;
```

Comment:
```suggestion
        total = 0;
```
зачем объявлять переменную, которая уже выше объявлена? какой смысл в этом?